lib/measured/base.rb

```diff
@@ -49,3 +47,6 @@ def method_missing(method, *args)
 require "measured/cache/json_writer"
 require "measured/cache/json"
 require "measured/measurable"
+require "measured/unit_error"
+require "measured/unit_already_added"
+require "measured/missing_conversion_path"
```

Comment:
I would hoist all these to the top of the require list here, at about line 39. Because if any initialization wants to raise or reference `Measured::UnitError` it cannot since it's the last thing to be loaded. It was the reason it was the first thing defined in the module.